packages/api/src/routes/medical/organization.ts

```diff
@@ -28,26 +32,30 @@ const router = Router();
  * Creates a new organization at Metriport and HIEs.
  *
  * @param req.body The data to create the organization.
+ * @param req.params.organizationType Optional organization type.
  * @returns The newly created organization.
  */
 router.post(
   "/",
   requestLogger,
   asyncHandler(async (req: Request, res: Response) => {
     const cxId = getCxIdOrFail(req);
-
+    const type = getFromQuery("organizationType", req);
```

Comment:
Damn, only now I realized we already had `OrgType` when we created the `OrganizationType`. What a mess. 😬 

What do you suggest we do here/there?

Reply:
Im going to use `OrganizationBizType` could use `OrganizationBusinessType` but i think shorthand gets the point across. Open to suggestions though

Reply:
Could also change the `OrgType` to like `TreatmentType` but we use this in a lot of places so as to limit the changes. I went with just updating the `OrganizationType`

Reply:
I like `TreatmentType`, maybe we can add a TODO and a `@deprecated` to it so next time we address it on an upstream PR?

Reply:
Let's also rename this param to `organizationBizType` so its consistent across the codebase?